wls-common/pom.xml

```diff
@@ -25,6 +25,7 @@
         <!-- Version muss mit der in den spring-boot-dependencies bereitgestellten Lombok-Version übereinstimmen -->
         <org.projectlombok.lombok.version>1.18.30</org.projectlombok.lombok.version>
         <org.projectlombok.mapstructbinding.version>0.2.0</org.projectlombok.mapstructbinding.version>
+        <org.mapstruct.version>1.4.2.Final</org.mapstruct.version>
```

Comment:
Aktuellste Version ist derzeit 1.5.5.Final - warum nehemn wir eine ältere?

Reply:
stimmt, wir sollten die latest version verwenden, solange es keinen Grund dagegen gibt. Habe entsprechend die aktuelle Version verwendet. Konnte in meinem Branch kein Problem damit feststellen